src/JackTrip.cpp

```diff
@@ -1289,14 +1289,7 @@ int JackTrip::clientPingToServerStart()
     // ----------------------------------------------
     connect(&mTcpClient, &QTcpSocket::readyRead, this, &JackTrip::receivedDataTCP);
     connect(&mTcpClient, &QTcpSocket::connected, this, &JackTrip::receivedConnectionTCP);
-    // Enable CI builds on Ubuntu 20.04 with Qt 5.12.8
-#ifdef __linux__
```

Comment:
This `ifdef` could potentially be changed into one based on a Qt version, but I guess now we are indeed going to draw a hard line for Qt 5.15, right?

Reply:
I thought about that after I did the change. But I mean, the vs gui is broken with 5.12.

Reply:
VS yes, but "novs" version could still be built on an older system with a version-based conditional.

Reply:
True. I'll change that.